precode.sh

```diff
@@ -6,38 +6,75 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p ./task/dir1
```

Comment:
Можно в одну строчку
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```